plugins/box-whiskers.js

```diff
@@ -27,27 +33,17 @@
                 var xs = screenModel.model.scaleX;
                 var ys = screenModel.model.scaleY;
                 var model = {
-                    x: function (d) {
```

Comment:
I saw some other plugins using ES6, but no problem. In the commit that I'll do later I'll move everything to ES5